miqa/core/models/annotation.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+from uuid import uuid4
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+
+class Decision(Enum):
+    NONE = '-'
+    GOOD = 'Good'
+    BAD = 'Bad'
+    USABLE_EXTRA = 'Usable extra'
+
+    @classmethod
+    def from_rating(cls, rating: str) -> str:
+        return {
+            '': cls.NONE.name,
+            '0': cls.BAD.name,
+            '1': cls.GOOD.name,
+            '2': cls.USABLE_EXTRA.name,
+        }[rating]
+
+    @classmethod
+    def to_rating(cls, decision: str) -> str:
+        return {
+            cls.NONE.name: '',
+            cls.BAD.name: '0',
+            cls.GOOD.name: '1',
+            cls.USABLE_EXTRA.name: '2',
+        }[decision]
+
+
+class Annotation(models.Model):
+
```

Comment:
This is looking good. Let's add `Meta.ordering = ['-created']` and add the composite index for our main query case:

```python
Meta.indexes = [
    models.Index(fields=['scan', '-created']),
]
```